src/vs/workbench/services/editor/browser/editorService.ts

```diff
@@ -1174,6 +1177,12 @@ export class EditorService extends Disposable implements EditorServiceImpl {
 
 	//#endregion
 
+	//#region activeEditorOutputModeChanged()
+	activeEditorOutputModeChanged(): void {
```

Comment:
You should try to avoid having to add this - the editor service should be generic across all editors where as the output mode is specific to query editors. 

At the very least this should probably go into the QueryEditorService, not here. But it'd be good to see if this kind of thing is even necessary - why do we need the global service to tell us when the output mode changes? The only thing that should generally care about that is the editor itself which can be notified directly.